src/components/Card.js

```diff
@@ -1,15 +1,17 @@
-import React from "react"
+import React from 'react'
 
-const Card = () => {
+const Card = ({hand}) => {
   return (
     <div className="handStyles">
-      {hand.map((card, index) => {
+      {hand.map((Card, index) => {
```

Comment:
Since the parameter of map is not a class we would want card to be in all lowercase letters.